### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Abdulwahab76**

### CODE REVIEW

File: src/features/dropdowns/index.tsx
Comment (abidali671) [inline src/features/dropdowns/index.tsx:1]: the file name should be `dropdowns.slice.ts`

File: src/store/index.ts [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import { configureStore } from '@reduxjs/toolkit';
+import dropDowns from 'src/features/dropdowns';
```
Comment (abidali671) [inline src/store/index.ts:2]: use path alias for import dropdowns, and it should be dropdownsReducer

Comment (abidali671) [inline src/store/index.ts:2]: use path alias for import dropdowns, and it should be dropdownsReducer

File: src/pages/auth/register/register.hook.ts [hunk 2]
```diff
@@ -10,9 +11,12 @@ const InitialSignUpValues = {
 };
 
 const useSign = () => {
```
Comment (abidali671) [inline src/pages/auth/register/register.hook.ts:13]: variable name should be `useRegister`

File: src/pages/addProduct/index.tsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
+import Container from '@components/container';
+import { Box, Stack, Typography, TextField, Checkbox } from '@mui/material';
+import { useSelector } from 'react-redux';
+
+import * as styles from './addproduct.styles';
+const AddProduct = () => {
+    const dropDown = useSelector((state) => state['dropdowns'].sizes);
```
Comment (abidali671) [inline src/pages/addProduct/index.tsx:7]: variable name should be sizes & useSelector callback return value should be `state.dropdowns.sizes`

Comment (Abdulwahab76) [reply src/pages/addProduct/index.tsx:7]: resolve

**Pull request merged**